### PR TITLE
Fix namespace and tableName validation in CorfuStoreBrowserMain.  Update README.

### PR DESCRIPTION
**Pull request by pankti-m**
Fix the validation of user-provided namespace and tablename in CorfuStoreBrowserMain.

Also update the README with sample usage of the editor command.

Related issue(s) (if applicable): #<number>

## Checklist (Definition of Done):

- [ ] There are no TODOs left in the code
- [ ] [Coding conventions](https://github.com/CorfuDB/CorfuDB/wiki/Corfu-Style-Guidelines) (e.g. for logging, unit tests) have been followed
- [ ] Change is covered by automated tests
- [ ] Public API has Javadoc

### CODE REVIEW

**Review by hisundar — APPROVED** [review]
LGTM

**Review by zhangn49 — APPROVED** [review]

File: corfudb-tools/README [hunk 1]
```diff
@@ -9,9 +9,9 @@ definitions or proto files to serialize/deserialize the data.
 --------
 Browser
 --------
-The CorfuStore Browser supports the following operations -
-The browser supports read operation which reads and outputs all entries in a
-given table and namespace, or list tables in a given namespace.
+The browser supports read operations which reads and outputs all entries in a
```
Comment (annym) [inline corfudb-tools/README:12]: a bit weird -> 'supports read operations which reads'... perhaps something like 'The browser allows to visualize/inspect corfu tables and metadata... by reading all entries...

File: corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserMain.java [hunk 2]
```diff
@@ -41,8 +41,9 @@
         "[--diskPath=<pathToTempDirForLargeTables>] "+
         "[--numItems=<numItems>] "+
         "[--batchSize=<itemsPerTransaction>] "+
-        "[--itemSize=<sizeOfEachRecordValue>] "+
-        "[--tlsEnabled=<tls_enabled>]\n"
+        "[--itemSize=<sizeOfEachRecordValue>] "
```
Comment (annym) [inline corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserMain.java:44]: Is it correct to think of renaming this file? as we are differentiating browser from editor, so its more like CorfuStoreBrowserEditorMain?

**Review by annym — APPROVED** [review]
Minor comments..

**Review by Lujie1996 — APPROVED** [review]

**Pull request merged**